audiorecorder

```diff
@@ -24,73 +28,113 @@ _lookup_devices(){
 _lookup_sample_rate(){
     case "${1}" in
         "44.1 kHz")
-        SAMPLE_RATE_NUMERIC="44100"
-        ;;
+            SAMPLE_RATE_NUMERIC="44100"
+            ;;
         "48 kHz")
-        SAMPLE_RATE_NUMERIC="48000"
-        ;;
+            SAMPLE_RATE_NUMERIC="48000"
+            ;;
         "96 kHz")
-        SAMPLE_RATE_NUMERIC="96000"
-        ;;
+            SAMPLE_RATE_NUMERIC="96000"
+            ;;
     esac
 }
 
 _lookup_bit_depth(){
     case "${1}" in
         "16 bit")
-        CODEC="pcm_s16le"
-        ;;
+            CODEC="pcm_s16le"
+            ;;
         "24 bit")
-       CODEC="pcm_s24le"
-        ;;
+            CODEC="pcm_s24le"
+            ;;
     esac
 }
+_get_ITEM_ID(){
+    echo "Please Input Item ID or type Q to exit"
+    read ITEM_ID
+    if [ "${ITEM_ID}" = "Q" ] || [ "${ITEM_ID}" = "q" ] ; then
```

Comment:
`if [[ "${ITEM_ID}" = [Qq] ]] ; then` would be a more compact alternative – just FYI :-)